README.md

```diff
@@ -98,14 +98,14 @@ Our audacious goal: _Zed will do for data lakes what JSON did for APIs._
 
 ## Zed & Brim
 
-Zed and Brim have become our _research vehicle_ for exploring the data model..
+Zed and Brim have become our _research vehicle_ for exploring the data model.
```

Comment:
There's plenty of `...` in the doc which is fine, but I wasn't sure which way `..` was meant to go, so I picked a side. :wink: